commands/features-tournaments.js

```diff
@@ -10,6 +10,14 @@ function tryGetRoomName (room) {
 }
 
 exports.commands = {
+	tierpoll: 'nexttour',
+	tourpoll: 'nexttour',
+	nexttour: function (arg, by, room, cmd) {
+		//This can be set to whatever you want
+		this.reply("/tierpoll"); //Placeholder unless anyone wants to set the format
+		this.reply("/poll timer 1.5"); //minor bug where if you do /tierpoll while a poll is going on it sets the timer
```

Comment:
I just don't think it would be beneficial as 1.5 minutes, seeing as small servers may never get many votes